modules/table/src/main/java/org/apache/ignite/internal/table/distributed/storage/InternalTableImpl.java

```diff
@@ -646,7 +647,7 @@ public CompletableFuture<Void> upsertAll(Collection<BinaryRowEx> rows, InternalT
                 rows,
                 tx,
                 this::upsertAllInternal,
-                InternalTableImpl::allOfResultFuture
+                RowBatch::allResultFuture
```

Comment:
It's either `allResultsFuture`, or `allResultFutures`, one of them must be plural

Reply:
Shit, my attentiveness is suffering!
Fix it.